eng/deploy.yaml

```diff
@@ -149,6 +149,32 @@ stages:
         enableCustomDeployment: true
         DeploymentType: webDeploy
         RemoveAdditionalFilesFlag: true
+  
+  - job: deployRolloutScorer
+    displayName: Deploy rollout scorer azure function
+    dependsOn: 
+    - approval
+    steps:
+    - download: current
```

Comment:
This should be Build since we moved to pipeline as resource

Reply:
ahh yes, good catch. thanks Sunanda!